email_mailbox/src/components/SideBarLabelItem.js

```diff
@@ -3,7 +3,11 @@ import PropTypes from 'prop-types';
 import './sidebarlabelitem.scss';
 
 const SideBarLabelItem = props => (
-  <li className="nav-item-label">
+  <li
+    className={
+      props.selected ? 'nav-item-label nav-item-selected' : 'nav-item-label'
+    }
+  >
```

Comment:
You could use ...
`<li className={*nav-item-label ${props.selected ? 'nav-item-selected' : ''}*} ` />
(Replace * to `)
... here?